tests/download/test_u_download_01_events.py

```diff
@@ -596,166 +595,18 @@ def test_get_events_errors(self,
             assert self.mock_urlopen.called
             self.mock_urlopen.reset_mock()
 
-    @patch('stream2segment.download.modules.events.isf2text_iter', side_effect=isf2text_iter)
-    def test_get_events_eventws_from_isc(self, mock_isf_to_text,
+    def test_get_events_eventws_from_isc(self,
                                          # fixtures:
                                          db, data):
         '''test bad events from isc'''
 
-        # normal query from emsc, data exopected as FDSN, returend as FDSN
+        # normal query from emsc, data expected as FDSN, returned as FDSN
         _ = self.get_events_df(None, db.session, 'emsc', {},
                                start=datetime(2010, 1, 1),
                                end=datetime(2011, 1, 1),
                                db_bufsize=self.db_buf_size)
-        assert not mock_isf_to_text.called
         assert db.session.query(Event.id).count() == 2
 
-        # data expected as isf, returned as FDSN:
-        with pytest.raises(FailedDownload) as fld:
-            # now it should raise because of a 413:
-            _ = self.get_events_df(None, db.session, 'isc', {},
```

Comment:
This line of code should not be removed in order to check that when we provide "isc", now we do NOT have exceptions (as the line above). Assert something like:
`assert db.session.query(Event.id).count() == 2` (2 might not be the number of events, just put whatever it is if the tests fail)